src/annotator/test/guest-test.js

```diff
@@ -788,10 +784,10 @@ describe('Guest', () => {
       ]);
     });
 
-    it('sets `$tag` to a falsey value', async () => {
+    it('sets `$tag` to a temporary value', async () => {
```

Comment:
"Temporary" is perhaps ambiguous here. It is "temporary" in the sense that it will change when the page is reloaded and the annotation is re-fetched from the server. It is not "temporary" in the sense that it won't change as long as the annotation is present in the page. Perhaps better to just say "sets $tag property".

Reply:
👍🏼